xseed.sh

```diff
@@ -20,12 +20,18 @@ elif [ -n "$sonarr_eventtype" ]; then
     filePath="$sonarr_series_path"
     folderPath="$sonarr_episodefile_sourcefolder"
     eventType="$sonarr_eventtype"
-elif [ -n "$Lidarr_EventType" ]; then
+elif [ -n "$lidarr_eventtype" ]; then
```

Comment:
This is an env var from lidarr; it cannot be modified to NOT match lidarr https://github.com/Lidarr/Lidarr/blob/b58f8e1e9f338e03d6336be7ce241b4bf233cce7/src/NzbDrone.Core/Notifications/CustomScript/CustomScript.cs#L94

Reply:
I went by their official wiki: https://wiki.servarr.com/lidarr/custom-scripts I totally understand your point though, it's fine for me either way.

Reply:
Wiki says the docs needs cleanup and links to the source :)